src/styles/all-styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
+}
```

Comment:
better make this block a somebodies element, because BEM block can have external styles (margin)